prime-router/src/main/kotlin/fhirengine/utils/HL7Reader.kt

```diff
@@ -21,198 +20,23 @@ import gov.cdc.prime.router.ActionLogger
 import gov.cdc.prime.router.InvalidReportMessage
 import org.apache.commons.lang3.exception.ExceptionUtils
 import org.apache.logging.log4j.Level
-import org.apache.logging.log4j.kotlin.Logging
+import org.apache.logging.log4j.kotlin.logger
 import java.util.Date
-import ca.uhn.hl7v2.model.v251.message.ORU_R01 as v251_ORU_R01
 import ca.uhn.hl7v2.model.v251.segment.MSH as v251_MSH
```

Comment:
Is there a way we can import only the `v27` version of MSH? I think this is another remnant of the old message model class behavior.

Reply:
Not if we want to keep the check that it is not v2.5

Reply:
Yep, I don't think we need that anymore. There shouldn't be a functional difference between the `v251` and `v27` versions of this, and we're actively looking to delete the `NIST` class entirely.

Reply:
For some reason I thought this was in a test. This is not in scope for the ticket.